doc/source/dev/reviewer_guidelines.rst

```diff
@@ -91,6 +91,10 @@ For maintainers
   explain why. If it's not, then it's a good idea to first explain why you
   think the PR is not suitable for inclusion in NumPy and then let a second
   committer comment or close.
+- If the PR submitter doesn't respond to your comments for 6 months, move the PR 
+  in question to the inactive category with the “inactive” tag attached. At this point,
+  the PR can be closed by a maintainer. If there is any interest in finalizing the PR under consideration, then this can be indicated at any time (without waiting 6 months) by a comment.   
```

Comment:
```suggestion
  the PR can be closed by a maintainer. If there is any interest in finalizing the PR under consideration,
  this can be indicated at any time, without waiting 6 months, by a comment.   
```